### DISCUSSION TITLE
评论收不到提醒邮件 || Not received a reminder email for comments

### DISCUSSION THREAD

**Discussion by 2176608** [walinejs/waline#2607] [Q&A] [answered]
### Describe the bug | 问题描述

可以正常发评论，但就是不能收到邮件提醒，密码配置正确，不是用博主邮箱来发评论的

### Website URL | 问题网站

paddi.us.kg

### Where your waline deploy? | 服务部署在哪里？

Vercel (Default)

### Where your comment data store? | 数据存储在哪里？

LeanCloud(https://leancloud.app)

---
###Describe the bug | Problem description

You can post comments normally, but you can't receive email reminders. The password is configured correctly, and you are not using the blogger's email address to post comments.

### Website URL | Problem Website

paddi.us.kg

### Where your waline deploy? | Where is the service deployed?

Vercel (Default)

### Where your comment data store? | Where is the data stored?

LeanCloud(https://leancloud.app)

**Comment by 2176608** [comment]
且微软账户未开启两步验证

---
And the Microsoft account does not have two-step verification enabled

**Comment by lizheming** [comment]
可以换个其他的邮箱服务，排查一下。如果还没有，可以去vercel后台看一下发布评论时候的接口日志看看。

---
You can change to another email service and check. If not, you can go to the vercel backend and take a look at the interface log when posting comments.

**Reply by 2176608** [reply]
![屏幕截图 2024-07-16 171318](https://github.com/user-attachments/assets/3a566e2a-b707-4cbb-9980-db05a541bf4c)
![屏幕截图 2024-07-16 171303](https://github.com/user-attachments/assets/8be1a64d-b8b5-4830-9446-13a88f138bd2)


---
![Screenshot 2024-07-16 171318](https://github.com/user-attachments/assets/3a566e2a-b707-4cbb-9980-db05a541bf4c)
![Screenshot 2024-07-16 171303](https://github.com/user-attachments/assets/8be1a64d-b8b5-4830-9446-13a88f138bd2)

**Reply by lizheming** [reply]
可以按照提示的错误信息自助解决下，看起来和 Waline 无关~

---
You can solve it by yourself according to the error message prompted. It seems to have nothing to do with Waline~

**Comment by 2176608** [comment, marked as answer]
Outlook邮箱本身问题

---
Problems with Outlook mailbox itself
